stablehlo/reference/Interpreter.cpp

```diff
@@ -97,6 +97,10 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       Tensor runtimeOperand = fetchOperand(floorOp.operand());
       Tensor runtimeResult = eval(floorOp, runtimeOperand);
       populateResults({runtimeResult});
+    } else if (auto negOp = dyn_cast<NegOp>(op)) {
+      Tensor runtimeOperand = fetchOperand(negOp.operand());
```

Comment:
Rebase this CL on main before submit, `operand()` will need to be updated to `getOperand()` with #203 submitted.

Reply:
Thanks for the heads up!